frontend/exporter/src/types/copied.rs

```diff
@@ -841,18 +861,65 @@ pub struct ExpnData {
 }
 
 /// Reflects [`rustc_span::Span`]
-#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema, Eq, Ord)]
+#[derive(::serde::Serialize, ::serde::Deserialize, Clone, Debug, JsonSchema, Eq, Ord)]
 pub struct Span {
     pub lo: Loc,
     pub hi: Loc,
     pub filename: FileName,
     /// Original rustc span; can be useful for reporting rustc
     /// diagnostics (this is used in Charon)
+    #[cfg(feature = "rustc")]
     #[serde(skip)]
     pub rust_span_data: Option<rustc_span::SpanData>,
+    #[cfg(not(feature = "rustc"))]
+    #[serde(skip)]
+    pub rust_span_data: Option<()>,
     // expn_backtrace: Vec<ExpnData>,
 }
 
+/// We need to define manual `impl`s of `Span`: we want to skip the
+/// field `rust_span_data`. The derive macros from `bincode` don't
+/// allow that, see https://github.com/bincode-org/bincode/issues/452.
```

Comment:
That's stupid. May be a reason to pick something other than bincode then at some point.

Reply:
:man_shrugging: let's postpone that anyway, that's only an internal format for communication between hax-driver and cargo-hax